islandora_pdf.info.yml

```diff
@@ -1,5 +1,5 @@
 name: 'Islandora PDF'
-description: 'A default Islandora Repository module to handle PDF files'
+description: 'A default Islandora Repository module to handle PDF files.'
```

Comment:
These descriptions have always struck me as problematic 'Islandora Repository' isn't a thing and how would an add on module be default? Perhaps something more along the lines of 'Adds PDF ingest and display functionality to Islandora'.